src/checkers/inference/VariableAnnotator.java

```diff
@@ -506,6 +506,15 @@ private Slot addPrimaryVariable(AnnotatedTypeMirror atm, final Tree tree) {
         } else {
             AnnotationLocation location = treeToLocation(tree);
             variable = replaceOrCreateEquivalentVarAnno(atm, tree, location);
+
+            // If a variable slot is created on any element of the anonymous constructor, it can't be
+            // inserted by AFU, so set it as uninsertable
```

Comment:
This isn't an issue with AFU - constructors in anonymous classes can't be made explicit. This is a restriction by the Java syntax, not a limitation of AFU.